core/src/avm2/globals/flash/display3D/context_3d.rs

```diff
@@ -298,3 +369,117 @@ pub fn clear<'gc>(
     }
     Ok(Value::Undefined)
 }
+
+pub fn create_texture<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    if let Some(context) = this.and_then(|this| this.as_context_3d()) {
+        // This is a native method, so all of the arguments have been checked and coerced for us
+        let width = args[0].as_integer(activation.context.gc_context)? as u32;
+        let height = args[1].as_integer(activation.context.gc_context)? as u32;
+        let format = args[2].coerce_to_string(activation)?;
+        let optimize_for_render_to_texture = args[3].coerce_to_boolean();
+        let streaming_levels = args[4].as_integer(activation.context.gc_context)? as u32;
+        let format = if &*format == b"bgra" {
+            Context3DTextureFormat::Bgra
+        } else {
+            panic!("Unsupported texture format: {:?}", format);
```

Comment:
Is this just for visibility?

Reply:
Yes - Stage3D support isn't complete enough yet for this to be a normal error.